README.md

```diff
@@ -25,20 +25,19 @@
 </p>
 
 ## Table of contents
-* [About](#About)
-* [Features](#Features)
-* [Installation & Configuration](#installation--configuration)
-  * [Docker method](#docker-method)
-* [Updating](#updating)
-  * [Docker method](#docker-method-1)
-* [License](#license)
-* [Contributing](#contributing)
-* [Developer documentation](#developer-documentation)
-  * [The development environment](#the-development-environment)
-    * [Set up](#set-up)
-    * [Usage](#usage)
-  * [Code quality](#code-quality)
-  * [Testing](#testing)
+- [Table of contents](#table-of-contents)
+- [About](#about)
+- [Features](#features)
+- [Installation & Configuration](#installation--configuration)
+  - [Docker method](#docker-method)
+- [Updating](#updating)
+  - [Docker method](#docker-method-1)
+- [License](#license)
+- [Contributing](#contributing)
+- [Developer documentation](#developer-documentation)
+  - [The development environment](#the-development-environment)
+  - [Code quality](#code-quality)
+  - [Testing](#testing)
 
```

Comment:
1. Why did the `*` list item prefix become `-`?
2. Why did the `h3`s in the "The development environment" section disappear?

Reply:
My bad, the file got auto-formatted when I saved it. 
I'll revert this changes